docker/run_tests.sh

```diff
@@ -43,15 +43,6 @@ $WEBPACK_BINARY
 $PYTHON manage.py collectstatic -v0 --noinput
 
 
-echo ""
-echo "--------------------------------------------------------------------------------------------"
-echo "Running JavaScript tests"
-$NPM test
-pushd frontend
-yarn test --watchAll=false
-popd
-
-
```

Comment:
Seems like this means we can no longer run the entire test suite with one `make` command as [documented](https://github.com/mozilla/pontoon/blob/3ff1829fc20129df076401da39fe90d9adcc3304/CONTRIBUTING.rst#running-tests)?

Reply:
Good point. Separated `make test-webapp` from `make test`, which now also runs the `frontend` tests.